lib/sshutils/authority.go

```diff
@@ -38,9 +39,15 @@ func GetCheckers(ca types.CertAuthority) ([]ssh.PublicKey, error) {
 
 // GetSigners returns a list of signers that could be used to sign keys.
 func GetSigners(ca types.CertAuthority) ([]ssh.Signer, error) {
-	out := make([]ssh.Signer, 0, len(ca.GetSigningKeys()))
-	for _, keyBytes := range ca.GetSigningKeys() {
-		signer, err := ssh.ParsePrivateKey(keyBytes)
+	keys := ca.GetActiveKeys().SSH
+	out := make([]ssh.Signer, 0, len(keys))
+	for _, kp := range keys {
+		// PrivateKeys may be missing when loaded for use outside of the auth
```

Comment:
If it's missing, presumably it will be missing for every key pair and the returned signers list will be empty, along with nil error? Will the caller handle this correctly?

Reply:
The only caller of this function is `services.ValidateCertAuthority`, so the returned signers are unused.
I renamed this function to `ValidateSigners` and changed it to only do the key parsing.